src/components/Bracket/Bracket.styles.ts

```diff
@@ -11,8 +11,8 @@ export const colors = {
 };
 export type ColorType = keyof typeof colors;
 
-export const curve = (corner: CornerType, isSolid: boolean) => dcnb(
-  'su-w-[83%] su-h-full su-border-t-2 su-border-l-2 su-rounded-tl-full',
+export const curve = (corner: CornerType, isSolid: boolean) => cnb(
+  'su-w-[83%] su-h-full',
```

Comment:
Remove the dupe classes (it was unintentional) here since we aren't using dedupe anymore